packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -161,6 +162,12 @@ const LayoutHeader = (props) => {
       props.toggleTopMenu(event);
     }
   };
+
+  const handleOnMouseLeave = (e) => {
+    if (!props.isTopMenuOpen) {
+      e.target.blur();
+    }
+  };
```

Comment:
I'm not sure we need this. Isn't the `outline: none;` enough?

Reply:
> I'm not sure we need this. Isn't the `outline: none;` enough?

We still need the focused state when the menu is open.